{"name": "Test login with correct login credentials", "status": "broken", "statusDetails": {"message": "TypeError: cred_are_given() takes 0 positional arguments but 1 was given\n", "trace": "  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n"}, "steps": [{"name": "Given credentials are given", "status": "broken", "statusDetails": {"message": "TypeError: cred_are_given() takes 0 positional arguments but 1 was given\n", "trace": "  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n"}, "start": 1727203669420, "stop": 1727203669420}, {"name": "When we input correct credentials", "status": "skipped", "start": 1727203669420, "stop": 1727203669420}, {"name": "Then login will be successful", "status": "skipped", "start": 1727203669420, "stop": 1727203669420}, {"name": "And correct profile open", "status": "skipped", "start": 1727203669420, "stop": 1727203669420}], "start": 1727203669419, "stop": 1727203669420, "uuid": "6d6380b7-de4c-4eb9-a1dc-716224869a4e", "historyId": "e30aa676d68625a7f1afca1785c423d1", "fullName": "Facebook login functionality: Test login with correct login credentials", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "logincase"}, {"name": "tag", "value": "sanity"}, {"name": "feature", "value": "Facebook login functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}